code/modules/organs/organ_external.dm

```diff
@@ -872,19 +872,21 @@ Note that amputating the affected organ does in fact remove the infection from t
 	return (species.default_mutations.Find(mutation))
 
 
-/datum/organ/external/proc/release_restraints()
-	if(owner.handcuffed && body_part in list(ARM_LEFT, ARM_RIGHT, HAND_LEFT, HAND_RIGHT))
-		owner.visible_message(\
-			"\The [owner.handcuffed.name] falls off of [owner.name].",\
-			"\The [owner.handcuffed.name] falls off you.")
+/datum/organ/external/proc/release_restraints(var/toporbottom = 0)// 0:both, -1: legcuffs, 1: handcuffs
```

Comment:
macros are better than magic numbers

Reply:
Alright I can throw in some global defines for it.